lambdas/data-migration2/src/executions.ts

```diff
@@ -67,6 +68,8 @@ export const migrateExecutions = async (
   knex: Knex
 ): Promise<MigrationSummary> => {
   const executionsTable = envUtils.getRequiredEnvVar('ExecutionsTable', env);
+  const bucket = envUtils.getRequiredEnvVar('system_bucket', env);
+  const stackName = envUtils.getRequiredEnvVar('stackName', env);
```

Comment:
Side note/:ant: For the purposes of this PR this is fine given the nature of this code.  

I've been trying to avoid creating situations where we're implicitly relying on env variables deeper in the code for switchable configuration where we don't have to, as those are essentially >globals<.

We might consider putting this block in the main lambda function and then include these as a parameter/part of a params object in `migrateExecutions`, `migrateGranulesAndFiles`, and `migratePdrs` for example.  

Again, not *this* PR, but something to consider for maintenance reasons in the future/a future arch discussion.